internal/charmstore/search.go

```diff
@@ -86,7 +71,7 @@ func (s *Store) UpdateSearch(r *router.ResolvedURL) error {
 	if s.ES == nil || s.ES.Database == nil {
 		return nil
 	}
-	if deprecatedSeries[r.URL.Series] || r.Development {
+	if r.Development || r.URL.Series != "" && !series.Series[r.URL.Series].SearchIndex {
```

Comment:
Maybe I am missing something but shouldn't this be `|| r.URL.Series == ""`?
If so, how are the tests passing?